### PR TITLE
Use Repox as npm registry

### PR DESCRIPTION
**Pull request by sebastien-marichal**

### CODE REVIEW

**Review by pavel-mikula-sonarsource — CHANGES_REQUESTED** [review]
This was painful copy paste issue :(

File: README.md [hunk 1]
```diff
@@ -40,6 +40,16 @@ All actions should be consumed from these branches.
 
 ## Development notes
 
+### Setup Repox authentication token
+
+Add the following line in `.npmrc` in your HOME folder:
+
+```ini
+//repox.jfrog.io/artifactory/api/npm/npm/:_auth = <npm-repox-token>
```
[the remark this answers is not in the document]
Comment (sebastien-marichal) [reply README.md:48]: I am using this locally...

[the remark this answers is not in the document]
Comment (sebastien-marichal) [reply README.md]: `_auth` is for tokens that are base64 encoded while `_authToken` is not.

**Review by pavel-mikula-sonarsource — APPROVED** [review]
LGTM

**Pull request merged**